docs/elements.md

```diff
@@ -704,22 +704,22 @@ generation if two (or more) choices are identical.
+| `answers-name`                | string                                    | —        | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question.           |
+| `weight`                      | integer                                   | 1        | Weight to use when computing a weighted average score over elements.                                                                                                           |
+| `display`                     | "block", "inline", or "dropdown"          | "inline" | How to display the input field.                                                                                                                                                |
+| `number-answers`              | integer                                   | special  | The total number of answer choices to display. Defaults to displaying one correct answer and all incorrect answers.                                                            |
+| `sort`                        | "random", "ascend", "descend", or "fixed" | "random" | Order to display answer choices.                                                                                                                                               |
+| `hide-letter-keys`            | boolean                                   | false    | Hide the letter keys in the answer list, i.e., (a), (b), (c), etc.                                                                                                             |
+| `all-of-the-above`            | string                                    | `false`  | Add "All of the above" choice. See below for details.                                                                                                                          |
+| `none-of-the-above`           | string                                    | `false`  | Add "None of the above" choice. See below for details.                                                                                                                         |
+| `all-of-the-above-feedback`   | string                                    | —        | Helper text to be displayed to the student next to the `all-of-the-above` option after question is graded if this option has been selected by the student.                     |
+| `none-of-the-above-feedback`  | string                                    | —        | Helper text to be displayed to the student next to the `none-of-the-above` option after question is graded if this option has been selected by the student.                    |
+| `external-json`               | string                                    | special  | Optional path to a JSON file to load external answer choices from. Answer choices are stored as lists under "correct" and "incorrect" key names.                               |
```

Comment:
Since we talked about getting rid of this feature, later I will change the docs to remove these attributes and denote that it's deprecated in the controller file.

Reply:
Removed from docs in latest push.